app/mappings/Constraints.scala

```diff
@@ -56,55 +57,47 @@ trait Constraints {
     case _ => Invalid(countryError)
   })
 
-  private val emailRegex = """^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$""".r
+  val emailRegex = """^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$""".r
```

Comment:
Could you please move this on the top of the file where other vals are and also provide the type?

Reply:
This has been addressed.